pages/common/hadolint.md

```diff
@@ -0,0 +1,16 @@
+# hadolint
+
+> Lints Dockerfiles to help you build [best practice](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices) Docker images.
```

Comment:
Linter always try to send you to best practices. Maybe we can try this:
```suggestion
> Dockerfile linter.
> See also: <https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices>.
```
But, in a general way, we don't include more than 1 link in the pages. Another reviewer will comment about this.